Makefile

```diff
@@ -96,3 +96,5 @@ command-lines:
 	go build -ldflags ${LD_FLAGS} -o=${BIN_DIR}/vcancel ./cmd/cli/vcancel
 	go build -ldflags ${LD_FLAGS} -o=${BIN_DIR}/vresume ./cmd/cli/vresume
 	go build -ldflags ${LD_FLAGS} -o=${BIN_DIR}/vsuspend ./cmd/cli/vsuspend
+	go build -ldflags ${LD_FLAGS} -o=${BIN_DIR}/vjobs ./cmd/cli/vjobs
+	go build -ldflags ${LD_FLAGS} -o=${BIN_DIR}/vqueues ./cmd/cli/vqueues
```

Comment:
Why do we need so many cli?

Reply:
They are simliar to Slurm cli like `squeue`, `scancel`, so that Slurm user would get used to our cli more easliy.

Reply:
I donot think so. If you make each command a binary, it will definitely be inflated. And it will enforce users to install more tools and remember all.